.github/workflows/sync_whatsapp_attachments.yaml

```diff
@@ -0,0 +1,36 @@
+name: Sync Whatsapp Attachments
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    env:
+      SOURCE_PARAMS: ${{ secrets.TURN_ATTACHMENTS }}
+      SERVICE_ACCOUNT_KEY_RAW: ${{ secrets.SERVICE_ACCOUNT_KEY_RAW }}
+      SERVICE_ACCOUNT_KEY_ANALYTICS: ${{ secrets.SERVICE_ACCOUNT_KEY_ANALYTICS }}
+      TURN_HEADERS: ${{ secrets.TURN_AUTH }}
+      SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
+      RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
```

Comment:
should be able to set this to "v6", and to remove the "with:" block, since caching is enabled by default on github-hosted runners.